run/run.py

```diff
@@ -153,12 +153,15 @@ def main(platform_id, platform, args, config):
             if args.path:
                 command.insert(3, args.path)
         else:
-            command = [
+            command = [                    
                 'xvfb-run', '--auto-servernum',
                 './wpt', 'run',
                 platform['browser_name'],
             ]
 
+            if args.no-xvfb:
```

Comment:
`no-xvfb` is not a valid Python property name. The expression will be interpreted as `args.no - xvfb`. That's inappropriate because `args` does not have a property named `no`, there is no binding named `xvfb`, and the result of a subtraction operation is not relevant for this code path.

We'll want to reference the property named `no_xvfb`, but before we can do that, we have to define the command-line option first.